From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Wed, 22 Feb 2017 15:26:26 +0300
Subject: [ARCHEOLOGY] Convert boot_mode and sun8i machid tou-boot patch

> X-Git-Archeology: > recovered message: > Needs testing on A10, A20, ... with legacy kernel
> X-Git-Archeology: - Revision b64408698796b0df34068e255981f78f5f627ab3: https://github.com/armbian/build/commit/b64408698796b0df34068e255981f78f5f627ab3
> X-Git-Archeology:   Date: Wed, 22 Feb 2017 15:26:26 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Convert boot_mode and sun8i machid tou-boot patch
> X-Git-Archeology:
> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
> X-Git-Archeology: - Revision bc46fd509a50db4bcef014d79569b68287ecc19b: https://github.com/armbian/build/commit/bc46fd509a50db4bcef014d79569b68287ecc19b
> X-Git-Archeology:   Date: Tue, 13 Jun 2023 20:19:36 +0200
> X-Git-Archeology:   From: Gunjan Gupta <viraniac@gmail.com>
> X-Git-Archeology:   Subject: allwinner: bump u-boot to v2022.10
> X-Git-Archeology:
---
 include/configs/sun8i.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/sun8i.h b/include/configs/sun8i.h
index 111111111111..222222222222 100644
--- a/include/configs/sun8i.h
+++ b/include/configs/sun8i.h
@@ -10,4 +10,6 @@
 
 #include <configs/sunxi-common.h>
 
+#define CONFIG_MACH_TYPE	(0x1029)
+
 #endif /* __CONFIG_H */
-- 
Armbian

